### ISSUE TITLE
The source pre-training with VGG16

### ISSUE THREAD

**Issue by yihong-97** [LTTM/UDAclustering#2]
Hi, I want pre-training on the source domain (**VGG16: GTA5, SYNTHIA, ResNet101:GTA5, SYNTHIA**), but I am not sure about the run code (learning rate, iter ...), Can you tell me?

If can provide the pre-trained weights，it's better >_<

Thanks！

**Comment by marco-toldo** [comment]
Here you can find the pre-trained weights:
- [ResNet-GTA5](https://drive.google.com/file/d/14LnlicyizbbqEeeZhUE-uOwtMYZ4p5fr/view?usp=sharing)
- [VGG-GTA5](https://drive.google.com/file/d/1tNWBLA1vArKl-Z-tRxgtguqkR8AMKNN6/view?usp=sharing)
- [ResNet-SYNTHIA](https://drive.google.com/file/d/13wH8gju3YvlfJDP9BCjZn9gC73RwH2r7/view?usp=sharing)
- [VGG-SYNTHIA](https://drive.google.com/file/d/1EjejJvAhdjefCJK7tf5hYzzeburg-SGC/view?usp=sharing)

**Comment by yihong-97** [comment]
> Here you can find the pre-trained weights:
> 
> * [ResNet-GTA5](https://drive.google.com/file/d/14LnlicyizbbqEeeZhUE-uOwtMYZ4p5fr/view?usp=sharing)
> * [VGG-GTA5](https://drive.google.com/file/d/1tNWBLA1vArKl-Z-tRxgtguqkR8AMKNN6/view?usp=sharing)
> * [ResNet-SYNTHIA](https://drive.google.com/file/d/13wH8gju3YvlfJDP9BCjZn9gC73RwH2r7/view?usp=sharing)
> * [VGG-SYNTHIA](https://drive.google.com/file/d/1EjejJvAhdjefCJK7tf5hYzzeburg-SGC/view?usp=sharing)

Thank you very much!!!!
